### PR TITLE
update generate tag

### PR DESCRIPTION
**Pull request by rasheedamir**

### CODE REVIEW

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-6-eb7fc947`

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -114,9 +114,10 @@ jobs:
       # Generate tag for operator without "v"
       - name: Generate Operator Tag
         id: generate_operator_tag
-        uses: anothrNick/github-tag-action@1.26.0
+        uses: github-tag-action@1.32.0
```
Comment (MaazMS) [inline .github/workflows/push.yml]: uses: anothrNick/github-tag-action@1.26.0  and new is uses: github-tag-action@1.32.0 is  correct.

**Review by MaazMS — DISMISSED** [review]

File: .github/workflows/push.yml (hunk 1 above)
Comment (rasheedamir) [reply .github/workflows/push.yml]: ouch good catch

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -114,9 +114,10 @@ jobs:
       # Generate tag for operator without "v"
       - name: Generate Operator Tag
         id: generate_operator_tag
-        uses: anothrNick/github-tag-action@1.26.0
+        uses: anothrNick/github-tag-action@1.32.0
```
Comment (MaazMS) [inline .github/workflows/push.yml:117]: update github tag version 1.26.0 to 1.32.0

File: .github/workflows/push.yml [hunk 3]
```diff
@@ -173,15 +174,17 @@ jobs:
...
           git commit -m "[skip-ci] Update artifacts" -a
+          
       - name: Push changes
         uses: ad-m/github-push-action@master
         with:
           github_token: ${{ secrets.STAKATER_GITHUB_TOKEN }}
 
       - name: Push Latest Tag
-        uses: anothrNick/github-tag-action@1.26.0
+        uses: anothrNick/github-tag-action@1.32.0
```
Comment (MaazMS) [inline .github/workflows/push.yml:184]: update github tag version 1.26.0 to 1.32.0

**Review by MaazMS — APPROVED** [review]
update github tag version 1.26.0 to 1.32.0

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-6-25db42f6`

**Pull request merged**